connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceTaskTest.java

```diff
@@ -220,6 +222,8 @@ public void testMissingFile() throws InterruptedException {
         task.stop();
     }
 
+    @Test
+    @Disabled
```

Comment:
Why did we disable this test?

Reply:
Currently this task will retries indefinitely, and this task was ignored at first.

Reply:
This test is not great but I don't think disabling it is the way forward.
I suggest keeping it but maybe reducing the number of retries. We can't really assert `poll()` will _indefinitely_ return `null` and there's little value in spending 100 seconds on this test.

Reply:
Good idea, I set the value to 3 as a better compromise value.